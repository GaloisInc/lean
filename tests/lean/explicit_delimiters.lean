example p : false := trivial -- Error

def main proof : false := trivial -- Error

example (p : false) := trivial

def main' (proof : false) := trivial

example := trivial
