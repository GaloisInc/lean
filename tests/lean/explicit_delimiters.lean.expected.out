explicit_delimiters.lean:1:8: error: invalid binder declaration, delimiter/bracket expected (i.e., '(', '{', '[', '{{')
explicit_delimiters.lean:3:9: error: invalid binder declaration, delimiter/bracket expected (i.e., '(', '{', '[', '{{')
